config/application.rb

```diff
@@ -81,7 +81,9 @@ class Application < Rails::Application
     # guaranteed to be available for other initializers to use.
     config.conjur_config = Conjur::ConjurConfig.new
 
-    # Initialize metrics and clean existing data
-    Monitoring::Prometheus.setup
+    #Initialize metrics and clean existing data
+    if config.conjur_config.telemetry_enabled 
```

Comment:
Nice! Very straightforward

Reply:
Fly by comment. I'm wondering if it's a good idea to move all the prometheus/telemetry-setup related things to their own initializer